.azure/azure-pipelines.perf.yml

```diff
@@ -351,6 +351,34 @@ stages:
           extraArgs: -PGO
           failOnRegression: 0
 
+- ${{ if eq(parameters.winuser_xdp, true) }}:
+  - stage: perf_winuser_qtip
+    displayName: Performance Testing Windows (XDP and QTIP)
+    dependsOn:
+    - build_winuser_xdp
+    jobs:
+    - template: ./templates/run-performance.yml
+      parameters:
+        pool: MsQuic-Win-Perf
+        platform: windows
+        localTls: schannel
+        remoteTls: schannel
+        iterations: ${{ parameters.iterations }}
+        arch: ${{ parameters.arch }}
+        protocol: ${{ parameters.protocol }}
+        logProfile: ${{ parameters.logging }}
+        timeout: ${{ parameters.timeout }}
+        extraArtifactDir: '_QTIP'
+        extraTestArgs: -ExtraArtifactDir _QTIP -XDP -QTIP
```

Comment:
This won't work because you need to download the `_Xdp` artifacts, but if you change that, then it would upload the results over the normal XDP results.

Reply:
I _think_ for now, let's add a manual flag at the top of the file to enable QTIP and have that flag off by default. When enabled it can modify the XDP run to enable QTIP so we can get a look.